src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -345,8 +345,8 @@ public function symlink($originDir, $targetDir, $copyOnWindows = false)
     /**
      * Given an existing path, convert it to a path relative to a given starting path.
      *
-     * @param string $endPath   Absolute path of target
-     * @param string $startPath Absolute path where traversal begins
+     * @param string $endPath   Path of target
+     * @param string $startPath Path where traversal begins
```

Comment:
In some rare edge cases we may not distinguish a path from being relative or starting with a drive (it's perfectly valid to have a directory named `c:` on Unix filesystems).